osu.Desktop.Deploy/osu.Desktop.Deploy.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I don't like that this stuff is still coming up in our changes. Why is this happening?

Reply:
BOM changes?